src/ui/units/wizard/components/ValuesList/ValuesList.tsx

```diff
@@ -63,6 +63,7 @@ export interface Props {
     onChangeSelectedValue: (selectedValue: string | null, shouldClearPalette?: boolean) => void;
     extra?: ExtraSettings;
     distincts?: Record<string, string[]>;
+    isMultipleFieldsSupported?: boolean;
```

Comment:
Is there really an additional flag needed here? Can't we rely on the number of items in `items`?

Reply:
You are right, there is no needs in it. I've added separate prop (`sectionFields`) for this case, which must be used only when multiple fields in section supported.